docs/r3/identity_messaging_and_consensus.md

```diff
@@ -0,0 +1,15 @@
+# Appendix 1: Identity, Messaging and Consensus in Corda and EVM Networks
+
+In Corda there is a one-to-one relationship between nodes in the network and permissioned identities maintained by the network manager. The CorDapp run by a node will run flows which exchange messages with other nodes/identities via the Corda platform's peer-to-peer messaging protocol. A "peer" in a Corda network is a node, addressable via RPC messages sent to the internet-addressable gateway of a Corda cluster.
```

Comment:
Need to point out this is an Arch V1 view, the node relationship to ID is an implementation detail